client/src/main/java/io/obswebsocket/community/client/OBSRemoteControllerBuilder.java

```diff
@@ -1,35 +1,34 @@
 package io.obswebsocket.community.client;
 
+import java.util.function.Consumer;
+
+import org.eclipse.jetty.websocket.client.WebSocketClient;
+
 import io.obswebsocket.community.client.listener.lifecycle.LifecycleListenerBuilderFacade;
 import io.obswebsocket.community.client.listener.lifecycle.controller.ControllerLifecycleListenerBuilder;
 import io.obswebsocket.community.client.message.event.Event;
-import java.util.function.Consumer;
-import org.eclipse.jetty.websocket.client.WebSocketClient;
+import lombok.Getter;
 
 /**
  * The internal builder for creating ${@link OBSRemoteController} instances.
  */
 public class OBSRemoteControllerBuilder {
 
-  private ControllerLifecycleListenerBuilder controllerLifecycleListenerBuilder = new ControllerLifecycleListenerBuilder(this);
-  private OBSCommunicatorBuilder obsCommunicatorBuilder = new OBSCommunicatorBuilder();
-  private LifecycleListenerBuilderFacade lifecycleListenerBuilderFacade = new LifecycleListenerBuilderFacade(
-    this,
-    obsCommunicatorBuilder.lifecycle(),
-    controllerLifecycleListenerBuilder
+  private final ControllerLifecycleListenerBuilder controllerLifecycleListenerBuilder = new ControllerLifecycleListenerBuilder(this);
+  private final OBSCommunicatorBuilder obsCommunicatorBuilder = new OBSCommunicatorBuilder();
+  private final LifecycleListenerBuilderFacade lifecycleListenerBuilderFacade = new LifecycleListenerBuilderFacade(
+          this,
+          obsCommunicatorBuilder.lifecycle(),
+          controllerLifecycleListenerBuilder
   );
   private OBSCommunicator communicator;
 
-  private WebSocketClient webSocketClient = WEBSOCKET_CLIENT();
+  @Getter private final WebSocketClient webSocketClient = new WebSocketClient();
```

Comment:
This was a bit of a hack to make it fully configurable instead of adding the one property I 'needed'. I'm not entirely sure if the 'actually disconnect when disconnecting' fix means that the stoptimeout doesn't trigger anymore, but before that, disconnecting caused a 30s hang of the thread. The websocket has some other timeouts that are also set quite high.

Reply:
I'll let that as it is for now